docs/mvvm/source-gen/diagnostics/MVVMTK0011.md

```diff
@@ -0,0 +1,56 @@
+---
+title: MVVM Toolkit error MVVMTK0011
+author: Sergio0694
+description: MVVM Toolkit error MVVMTK0011
+keywords: community toolkit, dotnet, csharp, mvvm, net core, net standard, source generators
+dev_langs:
+  - csharp
+---
+
+# MVVM Toolkit error MVVMTK0011
+
+The `CanExecute` name in `[RelayCommand]` must refer to a compatible member (either a property or a method) to be used in a generated command.
+
+The following sample generates MVVMTK0011:
+
+```csharp
+using CommunityToolkit.Mvvm.Input;
+
+namespace MyApp;
+
+public partial class SampleViewModel
+{
+    private bool Foo { set { } }
+
+    // The target property "Foo" does not have a setter
```

Comment:
do you mean getter?

Reply:
Whoops! Fixed in b015ab207d1090d99807f5f38c41b71156cf9eb5.